docs/node/run-your-node/prerequisites/set-up-tee.mdx

```diff
@@ -304,19 +314,24 @@ Make sure you have the following installed on your system:
 * [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config).
 * [OpenSSL](https://www.openssl.org) development package.
 
-On Fedora, you can install all the above with:
+<Tabs>
+<TabItem value="Fedora">
 
-```
+```shell
 sudo dnf install gcc protobuf-compiler pkg-config openssl-devel
 ```
 
-On Ubuntu, you can install all the above with:
+</TabItem>
+<TabItem value="Ubuntu">
 
-```
+```shell
 sudo apt install gcc protobuf-compiler pkg-config libssl-dev
 ```
 
-### Install [Rust](https://www.rust-lang.org)
+</TabItem>
+</Tabs>
+
+#### Install [Rust](https://www.rust-lang.org)
```

Comment:
Should we skip the Rust/sgx-detect part and just use the attestation-tool which also checks whether RA works?